modules/images/webp-uploads/helper.php

```diff
@@ -144,9 +152,7 @@ function webp_uploads_generate_additional_image_source( $attachment_id, $image_s
 		$destination_file_name = $editor->generate_filename( $suffix, null, $extension[0] );
 	}
 
-	remove_filter( 'image_editor_output_format', 'webp_uploads_filter_image_editor_output_format', 10, 3 );
-	$image = $editor->save( $destination_file_name, $mime );
-	add_filter( 'image_editor_output_format', 'webp_uploads_filter_image_editor_output_format', 10, 3 );
+	$image = webp_uploads_check_multi_mime_support( $editor, $destination_file_name, $mime );
```

Comment:
This change shouldn't be needed. Our `image_editor_output_format` filter is only there for WordPress core to change the _default_ MIME type output format, which is only relevant when we generate _only_ WebP. The code here is for generating _additional_ MIME type output files, so for that the filter always needs to be temporarily removed, so that we don't try to map the additional MIME type output format as well.

Please revert this change.